src/ui/CategoryFacet/CategoryFacetSearch.ts

```diff
@@ -231,4 +235,8 @@ export class CategoryFacetSearch implements IFacetSearch {
       this.categoryFacet.root
     );
   }
+
+  private shouldPositionSearchResults() {
+    return this.facetSearchElement.searchResults != null && this.facetSearchElement.searchResults.parentElement == null;
```

Comment:
Small point but it would help readability if using a local variable to get:
```
const results = this.facetSearchElement.searchResults;
return results &&  results.parentElement == null;
```